src/utils/getRandomNumber.js

```diff
@@ -0,0 +1,27 @@
+import NumberValidator from './validators/NumberValidator.js';
+
+/**
+ * 주어진 범위 내에서 랜덤 숫자를 반환하는 함수
+ *
+ * @param {number} num1
+ * @param {number} num2
+ *
+ * @example
+ * // 0부터 10까지의 랜덤 숫자 반환
+ * getRandomNumber(0, 10);
+ *
+ * @example
+ * // 5부터 15까지의 랜덤 숫자 반환
+ * getRandomNumber(5, 15);
+ */
+function getRandomNumber(num1, num2) {
```

Comment:
```suggestion
function getRandomNumber(min, max) {
```

이렇게 표현해주면 조금 더 자연스러울 것 같네요!